README.md

```diff
@@ -20,39 +20,31 @@
 * Install MySQL via Homebrew:
   ```
   brew install mysql
-  brew services list
+  brew services start mysql # Start MySQL at login
   mysql -V
   mysqladmin -u root password # Set your root password
   ```
 
 * Create a user and database for our Wordpress site:
   ```
-  mysql> CREATE DATABASE pickee_wp_development;
-  Query OK, 1 row affected (0.03 sec)
-
-  mysql> CREATE USER pickee_wp_admin@localhost IDENTIFIED BY 'password';
-  Query OK, 0 rows affected (0.01 sec)
-
-  mysql> GRANT ALL PRIVILEGES ON pickee_wp_development.* TO pickee_wp_admin@localhost;
-  Query OK, 0 rows affected (0.01 sec)
-
-  mysql> FLUSH PRIVILEGES;
-  Query OK, 0 rows affected (0.01 sec)
-
-  mysql> exit
-  Bye
+  mysql -u root -p -e "
+  CREATE DATABASE pickee_wp_development;
+  CREATE USER pickee_wp_admin@localhost IDENTIFIED BY 'password';
+  GRANT ALL PRIVILEGES ON pickee_wp_development.* TO pickee_wp_admin@localhost;
+  FLUSH PRIVILEGES;"
   ```
 
 ### Install PHP
 
 * Install PHP via Homebrew:
   ```
-  brew install php70 --with-debug
-  php -v
-  php-fpm -v
+  brew tap homebrew/homebrew-php
```

Comment:
Homebrew/php will be [archived](https://github.com/Homebrew/homebrew-php/issues/4721) by 31st March 2018 and we will need to switch to homebrew/core.